pkg/node/platform_description.go

```diff
@@ -0,0 +1,48 @@
+package node
+
+import (
+	"strconv"
+)
+
+type PlatformDescription struct {
+	imageName     string
+	daemonsetName string
+	labeltag      string
+	checkVersion
+}
+
+type checkVersion func(version string) bool
+
+var (
+	platforms = map[string]*PlatformDescription{
+		"default": {
+			imageName:     "csi-baremetal-node",
+			daemonsetName: "csi-baremetal-node",
+			labeltag:      "default",
+			// default checkVersion returns false everytime to detect only specific platforms
+			checkVersion: func(version string) bool { return false },
+		},
+		"kernel-5.4": {
+			imageName:     "csi-baremetal-node-kernel-5.4",
+			daemonsetName: "csi-baremetal-node-kernel-5.4",
+			labeltag:      "kernel-5.4",
+			checkVersion:  isNewKernel,
+		},
+	}
+)
+
+// isNewKernel returns true if kernel version >= 5.4
+func isNewKernel(version string) bool {
```

Comment:
`isNewKernel` name will be not relevant soon when we add `6.0` for example.

Reply:
New signature `moreThan(version string, supported float64)`. Maybe when we support 6.0, we will need to add `between` or something else